compose/src/main/java/com/maplibre/compose/camera/CameraState.kt

```diff
@@ -28,6 +31,7 @@ sealed class CameraState : Parcelable {
       result = 31 * result + zoom.hashCode()
       result = 31 * result + pitch.hashCode()
       result = 31 * result + direction.hashCode()
+      result = 31 * result + motion.hashCode()
```

Comment:
😂 This is why I really wish we didn't have to do this. Manual hashCode implementations are SO brittle.